File: app/src/main/java/com/dev/won/listviewexample/CheckableListViewActivity.java
package com.dev.won.listviewexample;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ListView;

import com.dev.won.listviewexample.adpater.Adapter_CheckableListView;
import com.dev.won.listviewexample.data.Item_Checkable;

import java.util.ArrayList;
import java.util.List;

public class CheckableListViewActivity extends AppCompatActivity {
    ListView lv_checkable;

    List<Item_Checkable> list_checkable = new ArrayList<>();

    Adapter_CheckableListView adapter_checkableListView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_checkable_list);

        lv_checkable = findViewById(R.id.lv_checkable);

        adapter_checkableListView = new Adapter_CheckableListView(getApplicationContext(), list_checkable);

        for (int i=0; i<10; i++) {
            list_checkable.add(new Item_Checkable("title"));
        }

        lv_checkable.setAdapter(adapter_checkableListView);
    }
}

File: app/src/main/java/com/dev/won/listviewexample/CheckableListViewActivity.kt
package com.dev.won.listviewexample

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.dev.won.listviewexample.adpater.Adapter_CheckableListView
import com.dev.won.listviewexample.data.Item_Checkable
import kotlinx.android.synthetic.main.activity_checkable_list.*

class CheckableListViewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_checkable_list)

        val lists = ArrayList<Item_Checkable>()

        for (i in 0..5) {
            lists.add(Item_Checkable("Title", false))
        }

        val adapter_checkableListView = Adapter_CheckableListView(applicationContext, lists)

        lv_checkable.adapter = adapter_checkableListView
    }
}
